projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.html

```diff
@@ -9,13 +9,21 @@
         [isRowSelected]="tableService.isRowSelected(row)"
         [isExpanded]="isExpanded(rowIndex)"
```

Comment:
pleae go over every place on template you added a function and used generic or pre calcualted data

Reply:
Ok. here changed to pre calculated data.